app/menus/menu.js

```diff
@@ -0,0 +1,39 @@
+const _paths = require('../config/paths');
+const {accelerators} = require('../accelerators');
```

Comment:
Maybe submenus could require paths and accelerators directly when needed. I don't know. What is the advantage to do this here ?